src/domain/application/components/ApplicationMap.tsx

```diff
@@ -182,9 +187,11 @@ export default function ApplicationMap({
       ) {
         // If mofified feature is going over hanke feature, revert back to original geometry
         modifiedFeature.setGeometry(originalFeature.getGeometry());
+      } else {
+        setSelectedFeature(modifiedFeature as Feature<Geometry>);
```

Comment:
SonarCloud says that "This assertion is unnecessary since it does not change the type of the expression."

Reply:
Removed unneeded assertion